From b28c3ef4911415e0eaf7175ddc000053ea3d9914 Mon Sep 17 00:00:00 2001
From: Bo Shen <voice.shen@atmel.com>
Date: Fri, 6 Mar 2015 11:38:40 +0800
Subject: [PATCH] Board: at91sam9n12ek: correct the crystal option

The at91sam9n12ek board use 16MHz crystal, while not 12MHz crystal.

Signed-off-by: Bo Shen <voice.shen@atmel.com>
Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
---
 board/Config.in                     | 2 +-
 board/at91sam9n12ek/Config.in.board | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/Config.in b/board/Config.in
index a2cf9c0..bc9fd13 100644
--- a/board/Config.in
+++ b/board/Config.in
@@ -164,7 +164,7 @@ config CONFIG_CRYSTAL_12_000MHZ
 	depends on ALLOW_CRYSTAL_12_000MHZ
 
 config CONFIG_CRYSTAL_16_000MHZ
-	bool "Build for use of an 16.000 MHz crystal (not yet supported)"
+	bool "Build for use of an 16.000 MHz crystal"
 	depends on ALLOW_CRYSTAL_16_000MHZ
 
 config CONFIG_CRYSTAL_16_36766MHZ
diff --git a/board/at91sam9n12ek/Config.in.board b/board/at91sam9n12ek/Config.in.board
index 617aacf..9899bf5 100644
--- a/board/at91sam9n12ek/Config.in.board
+++ b/board/at91sam9n12ek/Config.in.board
@@ -7,7 +7,7 @@ config	CONFIG_AT91SAM9N12EK
 	select ALLOW_SDCARD
 	select ALLOW_HSMCI
 	select ALLOW_CPU_CLK_400MHZ
-	select ALLOW_CRYSTAL_12_000MHZ
+	select ALLOW_CRYSTAL_16_000MHZ
 	select ALLOW_BOOT_FROM_DATAFLASH_CS0
 	select ALLOW_DATAFLASH_RECOVERY
 	select ALLOW_NANDFLASH_RECOVERY
-- 
1.9.1

